content/outcomes/secure-software-supply-chain/building-a-container.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Building a Container
+weight: 2
+layout: single
+related: ["/guides/containers/cnb-gs-pack/", "/guides/containers/cnb-gs-kpack/"]
```

Comment:
This is not rendering correctly, not sure why.

Reply:
Looks like it was due to the trailing backslash, fixed in the latest commit